pkg/controller/restore_controller.go

```diff
@@ -531,10 +535,21 @@ func (c *restoreController) runRestore(
 	}
 	defer closeAndRemoveFile(resultsFile, c.logger)
 
+	pluginManager := c.newPluginManager(log)
```

Comment:
There's already a plugin manager instantiated for the restore at https://github.com/heptio/ark/pull/938/files#diff-7d210f9ced78ff8be1ba497bf49e1a40R199 so I'd pass that into the `runRestore` func and use it here instead of instantiating a new one.